### PR TITLE
Sprites: Extract sprites from PIC.DLL

### PR DESCRIPTION
**Pull request by sotiredofyours**
Part of #21

### CODE REVIEW

File: O21.Game/U95/Sprites.fs [hunk 1]
```diff
@@ -60,13 +62,36 @@ module Sprites =
         Array.init backgroundGraphics.Length (fun i ->
             createSprite device backgroundGraphics[i] backgroundGraphics[i]
         )
-         
+        
+    // TODO: calculate indexes
```
Comment (ForNeVeR) [inline O21.Game/U95/Sprites.fs:66]: Could you please clarify and/or implement this TODO in this PR?

File: O21.Resources/Dib.fs [hunk 2]
```diff
@@ -33,7 +33,15 @@ type Dib(dib: byte[]) =
                                              
     member _.GetPixel(x: int, y: int): RGB =
         let y = height - y - 1 // turn the image upside-down
-        if colorDepth = 4us then
+        if colorDepth = 1us then
```
Comment (ForNeVeR) [inline O21.Resources/Dib.fs:36]: Ok, are there really fish with color depth = 1? Or is it for transparency masks?

**Review by ForNeVeR — CHANGES_REQUESTED** [review]

File: O21.Resources/Dib.fs (hunk 2 above)
Comment (sotiredofyours) [reply O21.Resources/Dib.fs:36]: Yes, all transparency masks in this dll have a color depth = 1. The color palette here contains only two colors: black (0,0,0) or white (255,255,255).

File: O21.Game/U95/Sprites.fs (hunk 1 above)
Comment (sotiredofyours) [reply O21.Game/U95/Sprites.fs:66]: This is about the order in which the bitmaps are stored in dll file: 45 transparency masks for left direction-> 45 graphics bitmaps and same for other direction. I will Implement it in next commit.

**Review by ForNeVeR — APPROVED** [review]
Thanks!

**Pull request merged**